packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -659,6 +659,32 @@ class WebViewController {
   Future<String> getTitle() {
     return _webViewPlatformController.getTitle();
   }
+
+  /// Set the scrolled position of this view.
+  /// 'x' - the x position to scroll to.
```

Comment:
What is the unit for `x` and `y`? is it the same unit across platforms?

Reply:
In the coming commit, I adjusted the comment to state pixel specifically. 
I was not sure at first iOS also works in pixels but looking into it, the "point" unit they require eventually translates into pixel.

In Apple's official docs they name the unit as 'Point' and the value is translated using CGFloat but other articles proclaim that Apple rounds these points since graphical rendering cannot split pixels. Eventually it means iOS is using pixels as well.

**Sources:**
- https://developer.apple.com/documentation/uikit/uiscrollview/1619404-contentoffset
- http://andrewmarinov.com/uiscrollviews-contentoffset-precision/

**Additionally it looks like common working practice is to treat this input as pixels:**
- https://stackoverflow.com/questions/3719753/iphone-uiscrollview-speed-check
- https://stackoverflow.com/questions/48319873/uiscrollview-resting-at-incorrect-contentoffset-20-px-not-0px

Reply:
Thanks!
We should emphasize that that these are pixels in the webview space (e.g these are not Flutter's logical pixels)

Reply:
Hey Amir, I'm not sure what the best phrasing is, suggesting:
"the x position, in pixels, to scroll the WebView content to."
Same for y position?

Reply:
What do you think about:

```dart
/// Sets the webview's content scroll position.
///
/// The parameters `x` and `y` specify the scroll position in webview pixels.
```

Reply:
100%, Added in incoming commit.